### ISSUE TITLE
[A11Y] Navigation components missing descriptive text (aria-label)

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#13327]
## Issue Description
Navigation items (excluding the skip links menu) throughout the application lack `aria-label` attributes. This omission prevents screen readers from providing meaningful context for each navigation menu, significantly impacting accessibility for users relying on assistive technologies. Users will hear "navigation" but not further meaning from each element.

## How to reproduce / Current behavior
1. Navigate to any page within the OJS installation using a screen reader (e.g., NVDA, JAWS, or VoiceOver).
2. Activate the navigation menu.
3. Observe that while skip links function correctly, subsequent navigation elements do not announce their specific name or purpose.

## What application are you using?
OJS version 3.5

## Stack used 
- Device: Desktop/Laptop
- OS: macOS
- Browser: Edge

## Testing method
- Method: Manual testing with screen reader and keyboard navigation

## Action
Add unique, descriptive `aria-label` attributes to each navigation components, i.e., `<nav></nav>` tag.
It is recommend the following labels:
- Main menu top
- Login and user account top
- Institutional
- Main menu bottom 
- Login and user account bottom

## Additional information
Screenshots showing the navigation elements in the page:

<img width="643" height="101" alt="top area with navigation components missingg aria-label attribute" src="https://github.com/user-attachments/assets/1dc62b28-1c4a-4ed8-9188-1a73061be081" />

<img width="608" height="166" alt="bottom area with navigation components missingg aria-label attribute" src="https://github.com/user-attachments/assets/3c818a19-ecd5-48f6-a806-b05619a42dd3" />
